util/fibers/listener_interface.cc

```diff
@@ -348,7 +348,7 @@ uint32_t ListenerInterface::GetMaxClients() const {
 }
 
 void Connection::Shutdown() {
-  auto ec = socket_->Shutdown(SHUT_RDWR);
+  auto ec = CHECK_NOTNULL(socket_)->Shutdown(SHUT_RDWR);
```

Comment:
Just making sure - this is unrelated to the third_pary.cmake changes you've made above right?

Reply:
sorry, upodated the PR description. merging